### ISSUE TITLE
Getting error when provoding starting time

### ISSUE THREAD

**Issue by EivindB** [codegouvfr/eleventy-plugin-calendar#4]
Hi, thank you for this great plugin, it's what my little community website was looking for to provide a subscribeable calendar for our members to see what events are coming up without visiting the website.

I have however gotten some trouble getting the exact start-time to generate as an ICS file, it handles all-day events just great, but it would be good to have exact times if possible.

Using the example in the documentation: `2016-05-25T09:24` I get this message: `[11ty]   TypeError: Invalid or missing event start date. (via Template render error`.

I'm not skilled enough to find a fix for it myself, so I thought I'd report it to you that you could check if you get the same error on your end with a known, functioning setup.

**Comment by hjonin** [comment]
Hi @EivindB !

Could you provide me with the frontmatter of your file which is not working so I can check the situation on my side?

Thanks in advance! :)

**Comment by EivindB** [comment]
Sure thing, here's some source to perhaps shed some light on the issue:

**the-event-title.md**
```the-event-title.md
layout: event.njk
tags: hending
title: Bygdedyret
start: 2024-11-28
duration:
  hours: 4
description: "Ein lokal Detaljar finn du her: [Nettsida](https://url.no/)"
```

This frontmatter generates just fine and results in the following ics.
**event.ics**
```
BEGIN:VCALENDAR
VERSION:2.0
CALSCALE:GREGORIAN
PRODID:adamgibbons/ics
METHOD:PUBLISH
X-PUBLISHED-TTL:PT1H
BEGIN:VEVENT
UID:R2xt4h7uH4odtn4eJg4_1
SUMMARY:Bygdedyret
DTSTAMP:20241206T201148Z
DTSTART;VALUE=DATE:20241128
DTEND;VALUE=DATE:20241129
DESCRIPTION:Ein lokal Detaljar finn du her: [Nettsida](https://url.no/)
LOCATION:thePlace
ORGANIZER;CN=&quot;theName&quot;:MAILTO:e@Mail.com
END:VEVENT
END:VCALENDAR
```

Hope it helps. I use some norwegian wording in there but it should map fine to the configuration described in the repo.

**Comment by hjonin** [comment]
Hi @EivindB !

Thanks for those details.

The problem was a bug in date parsing (11ty sometimes pass it as an object, some other times as a string - maybe since 11ty v.3 only). It should be fixed in the [`3.0.6`](https://www.npmjs.com/package/@codegouvfr/eleventy-plugin-calendar/v/3.0.6) version. :slightly_smiling_face:

**Comment by EivindB** [comment]
I've updated to the latest version, and the time is working! Thank you for such a swift fix!
